src/scheduler/parse.cpp

```diff
@@ -876,6 +876,15 @@ init_config()
 			free(conf.dynamic_res[i].script_name);
 		}
 	}
+	/* clear peer queues */
+	if (conf.peer_queues[0].local_queue != NULL) {
+		int i;
+		for (i = 0; i < NUM_PEERS; i++) {
```

Comment:
you can declare `i` in the for loop if you want c:

Reply:
creature of habit :)